nims-notes-things/src/components/Form.js

```diff
@@ -34,16 +30,11 @@ const Form = (props) => {
   };
 
   const validateSubmit = () => {
-    let isValidated = true;
+    let isValidated = false;
```

Comment:
confused on logic here, why the change?

Reply:
Alex went through the userRef - so i've implemented that now :)